flake.nix

```diff
@@ -318,19 +321,12 @@
     slinky-src.url = "github:skip-mev/slinky/v0.2.0";
     slinky-src.flake = false;
 
-    rollapp-evm-src.url = "github:dymensionxyz/rollapp-evm/21b29f6e77f5c11a2036252d60819810abbbd7b8";
-    rollapp-evm-src.flake = false;
-
```

Comment:
why is `rollup-evm` removed?

Reply:
If needed I can restore it, but I will need to update the version to a more recent one had error after updating `nixpkgs`

Reply:
I guess, it's fine as along as this is communicated to the users of `cosmos.nix`.